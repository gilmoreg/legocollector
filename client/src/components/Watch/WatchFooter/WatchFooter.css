.WatchFooter button {
    display: block;
    background-color: rgba(255,246,229,0.7);
    border-top: none;
    border-radius: 0px 0px 5px 5px;
    max-width: 760px;
    width: 100%;
    margin: 0 auto;
    z-index: -2;
}