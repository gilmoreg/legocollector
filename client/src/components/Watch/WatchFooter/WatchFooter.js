import React from 'react';
import PropTypes from 'prop-types';

import './WatchFooter.css';

const WatchFooter = props => (
  <div className="WatchFooter">
    <button onClick={props.click}>
			{ props.collapsed ?
				<i class="fa fa-angle-double-down" aria-hidden="true"></i>
				:
				<i class="fa fa-angle-double-up" aria-hidden="true"></i>
			}
		</button>
  </div>
);

WatchFooter.propTypes = {
	click: PropTypes.func.isRequired,
	collapsed: PropTypes.bool.isRequired,
}

export default WatchFooter;
