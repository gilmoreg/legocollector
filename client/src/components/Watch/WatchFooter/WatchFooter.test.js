import React from 'react';
import { shallow } from 'enzyme';
import WatchFooter from './WatchFooter';

it('renders without crashing', () => {
  const wrapper = shallow(
    <WatchFooter click={()=>{}}/>
  );
  expect(wrapper).toMatchSnapshot();
});
