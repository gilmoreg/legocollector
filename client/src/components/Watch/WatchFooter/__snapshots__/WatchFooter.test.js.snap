// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WatchFooter
    click={[Function]}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": <button
        onClick={[Function]}
      >
        <i
          aria-hidden="true"
          class="fa fa-angle-double-up"
        />
      </button>,
      "className": "WatchFooter",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": <i
          aria-hidden="true"
          class="fa fa-angle-double-up"
        />,
        "onClick": [Function],
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "aria-hidden": "true",
          "class": "fa fa-angle-double-up",
        },
        "ref": null,
        "rendered": null,
        "type": "i",
      },
      "type": "button",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": <button
          onClick={[Function]}
        >
          <i
            aria-hidden="true"
            class="fa fa-angle-double-up"
          />
        </button>,
        "className": "WatchFooter",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": <i
            aria-hidden="true"
            class="fa fa-angle-double-up"
          />,
          "onClick": [Function],
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "aria-hidden": "true",
            "class": "fa fa-angle-double-up",
          },
          "ref": null,
          "rendered": null,
          "type": "i",
        },
        "type": "button",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
